_api-reference/ingest-apis/processors-reference/remove.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Remove
+parent: Ingest processors 
+grand_parent: Ingest APIs
+nav_order: 230
+---
+
+# Remove
+
+The remove processor is used to remove a field from a document. The syntax for the `remove` processor is: 
+
+```json
+{
+    "remove": {
+        "field": "field_name"
+    }
+}
+```
+
+#### Configuration parameters
+
+The following table lists the required and optional parameters for the `remove` processor.
+
+| Name  | Required  | Description  |
+|---|---|---|
+| `field`  | Required  | Specifies the name of the field that you want to remove. |
+| `ignore_missing`  | Optional  | Specifies whether the processor should ignore documents that do not have the specified field. Default is `false`.  |
+| `ignore_failure`  | Optional  |  Specifies whether the processor should continue processing documents even if it fails to remove the specified field. Default is `false`.  |
+| `if`  | Optional  | Conditionally deploys the processor based on the value of the field. The `value` parameter specifies the value that you want to compare the field to. |
+| `tag`  | Optional  | Allows you to identify the processor for debugging and metrics.  |
+`description`  | Optional  | Brief description of the processor.  |  
+
+
+Following is an example of an ingest pipeline using the `remove` processor. 
+
+```json
+PUT /_ingest/pipeline/remove_ip
+{
+  "description": "Pipeline that excludes the ip_address field.",
+  "processors": [
+    {
+      "remove": {
+        "field": "ip_address"
+      }
+    }
+  ]
+}
+
+PUT testindex1/_doc/1?pipeline=remove_ip
```

Comment:
To make it clearer that it doesn't delete the whole document, I would index a doc with 2 fields:
```
PUT testindex1/_doc/1?pipeline=remove_ip
{
  "ip_address": "203.0.113.1",
  "name": "John Doe"
}
```